pkgs/kde/gear/kdeconnect-kde/default.nix

```diff
@@ -11,6 +14,19 @@
 mkKdeDerivation {
   pname = "kdeconnect-kde";
 
+  patches = [
+    (substituteAll {
+      src = ./hardcode-sshfs-path.patch;
+      sshfs = lib.getExe sshfs;
+    })
+  ];
+
+  # Hardcoded as a QString, which is UTF-16 so Nix can't pick it up automatically
+  postFixup = ''
+    mkdir -p $out/nix-support
+    echo "${sshfs}" > $out/nix-support/depends
```

Comment:
```suggestion
    echo "${sshfs}" >> $out/nix-support/depends
```

Reply:
That file isn't real and is not created by anything except us.

Reply:
ah okay, I thought that was somehow also a special nix file.